.drone.star

```diff
@@ -2,6 +2,7 @@ def main(ctx):
   versions = [
     'latest',
     'fedora',
+    'qt67'
```

Comment:
these are linux distros (`latest->ubuntu` but we are not using it because of https://github.com/owncloud-ci/squish/issues/34)
```suggestion
```